main.py

```diff
@@ -25,19 +30,28 @@
     boxes = face_recognition.face_locations(frame)
     encodings = face_recognition.face_encodings(frame, boxes)
     matches = []
-    for e,encoding in enumerate(encodings):
-        matches = face_recognition.compare_faces(Knownencodings,encoding)
+    foundnames = []
+    for x,encoding in enumerate(encodings):
+        matches = face_recognition.compare_faces(Knownencodings,encoding, tolerance = TOLERANCE)
```

Comment:
`tolerance = TOLERANCE` doesn't seem correct use here. Do you want to use TOLERANCE?